.github/workflows/hugo_ci.yaml

```diff
@@ -1,6 +1,10 @@
 name: Hugo Build and Deploy
 
-on: push
+on: 
+  pull_request:
+    types: [opened, reopened, closed]
+    branches: 
```

Comment:
I don't think we need to specify branch in here?

Reply:
That was based on @michaelfitzo's comment about PRs targeting main/master. Probably overzealous, but didn't want to cause confusion by pushing changes merged to a feature branch to alpha.gen3.org.